src/modules/launcher/PowerLauncher/Helper/EnvironmentHelper.cs

```diff
@@ -5,51 +5,68 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.ComponentModel;
-using System.Diagnostics;
-using System.Linq;
 using System.Security.Principal;
-using Microsoft.Win32.SafeHandles;
 using Wox.Plugin.Logger;
+using Stopwatch = Wox.Infrastructure.Stopwatch;
 
 namespace PowerLauncher.Helper
 {
     public static class EnvironmentHelper
     {
-        private const string Username = "USERNAME";
-        private const string ProcessorArchitecture = "PROCESSOR_ARCHITECTURE";
-        private const string Path = "PATH";
+        private const string PathVariable = "PATH";
+        private static HashSet<string> protectedProcessVariables;
```

Comment:
@jaimecbernardo 
Here I get the compiler warning `CS0649: Field 'field' is never assigned to, and will always have its default value 'value'`. My problem is that I can't suppress it. Do you have an idea how to fix suppress this.

Reply:
You want to initialize it to something, or it will always be null, right?
`private static HashSet<string> protectedProcessVariables = new HashSet<string>();`

Reply:
> You want to initialize it to something, or it will always be null, right?

Not sure what you want to say. My plan is to initialize an empty HashSet here which is privat and can only be changed by the public method.

Reply:
You should initialize it like this, I think:
`private static HashSet<string> protectedProcessVariables = new HashSet<string>();`